archivist/access_policies.py

```diff
@@ -27,7 +27,7 @@
 
 # pylint:disable=unused-import      # To prevent cyclical import errors forward referencing is used
 # pylint:disable=cyclic-import      # but pylint doesn't understand this feature
-import archivist as type_helper
+from archivist import archivist as type_helper
```

Comment:
hmmm - on looking at this again I wonder if

      from     .   import    archivist as type_helper

might be better as it forces local import (on the offchance that archivist is an installed pkg somewhere...)

Reply:
Hmm I swear I did that yesterday